torchao/csrc/cuda/fp6_llm/ptx_mma.cuh

```diff
@@ -38,9 +38,9 @@
 
 #ifdef PIPELINE_LEVEL_SMEM
 template <typename TilingConfig>
-__device__ __forceinline__ void B_FromSharedToReg(uint32_t  __restrict__    Reg[][4],
-                                                  half      __restrict__    (*read_SPTR)[WARP_K+PADDING_SHARED_MEM_FOR_B_8],
-                                                  int                       slice_id) {
+__device__ __forceinline__ void B_FromSharedToReg(uint32_t Reg[][4],
+                                                  half     (*read_SPTR)[WARP_K+PADDING_SHARED_MEM_FOR_B_8],
+                                                  int      slice_id) {
```

Comment:
Missing some `__restrict__` here.
For `Reg[][4]`, I don't know if we can add `__restrict__` directly. Otherwise, maybe we need to change it to a pointer (so we can add back `__restrict__`). From what I know, `Reg[][4]` is still passed as pointer, but it allows us to do 2d-indexing (last dim is compile-time constant, so it translates to `4 * first_index + second_index`).